Prototype build

JohnByrneJames opened the pull request:
Changes

JohnByrneJames committed:
Changes to prototype

This is working needs small changes that I plan to do tomorrow before the lecture. But the UI is a little tricky for me to work with. Doesn't work as it usually does - over to you neil.

JohnByrneJames committed:
Integration Change

Small change

Rokkii reviewed (dismissed):
Profanity aside xD  LGTM

RubyNova commented on UnityClient/Assets/Scripts/AStar/Graph.cs:
```diff
@@ -201,6 +202,12 @@ public Node NodeFromWorldPoint(Vector3 worldPosition)
         float percentX = (worldPosition.x + gridWorldSize.x / 2) / gridWorldSize.x;
         float percentY = (worldPosition.z + gridWorldSize.y / 2) / gridWorldSize.y;
+
+/*        if (percentX > 1 || percentX < 0 || percentY > 1 || percentY < 0)
```
This is dead code, should delete this probably. Lol.

RubyNova commented on UnityClient/Assets/Scripts/AStar/Pathfinding.cs:
```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using System;
 using UnityEngine;
+using Debug = UnityEngine.Debug;
```
Why the alias?

RubyNova commented on UnityClient/Assets/Scripts/AStar/Pathfinding.cs:
```diff
@@ -24,6 +25,7 @@ public void StartFindPath(Vector3 startPos, Vector3 targetPos)
 
     IEnumerator FindPath(Vector3 startPos, Vector3 targetPos)
```
Accessor is missing. Please provide an accessor.

JohnByrneJames committed:
Changes Requested

The debug part was not working before I did that change, it was a while ago I did that.

RubyNova reviewed (approved):
verly loosely LGTM. Further refactoring should be propagated to another PR.
